content/en/docs/Overview/_index.md

```diff
@@ -7,3 +7,40 @@ description: >
 ---
 
 Apache Parquet is a columnar storage format available to any project in the Hadoop ecosystem, regardless of the choice of data processing framework, data model or programming language.
+
+This documentation contains information about both the [parquet-mr](https://github.com/apache/parquet-mr) and [parquet-format](https://github.com/apache/parquet-format) projects. 
+
+
+### Parquet Format
+
+The "Parquet Format" project hosts the official specification of the Parquet file format, defining how data is structured and stored. This specification, along with Thrift metadata definitions and other crucial components, is essential for developers to effectively read and write Parquet files. The parquet-format project specifically contains the format specifications needed to understand and properly utilize Parquet files.
```

Comment:
Let's please make spelling of the project name consistent: `parquet-format` not "Parquet Format".

Reply:
Also, I would write "understand, read and write Parquet files" rather than "understand and properly utilize".